src/tex.js

```diff
@@ -57,6 +57,34 @@ MathJax.Extension.Arabic = {
       '8': '٨',
       '9': '٩'
     },
+    // numbers with arabic decimal separator ','
+    arabicDecimalSeparator: {
```

Comment:
I don't think we need this map, we can use RegExp instead without the need to repeat what's in `numbersMap`. I can add more details if you'd like.